README.md

```diff
@@ -45,28 +45,32 @@ For an existing Angular application,
 
 3. **Import the modules you want to use.**
 
-    To add the entire library, add
+    To add the entire library, add the following code to your main application module.
 
     ```javascript
     import { FundamentalNgxModule } from 'fundamental-ngx';
+    
+    @NgModule({
+        imports: [FundamentalNgxModule],
```

Comment:
does this mean they have to replace their `@NgModule` or just add in the imports?

Reply:
Seems obvious to me that it just requires adding the imports. Is it not? I could add some ellipsis to symbolize other stuff around the module import